### PR TITLE
Fix travis build

### PR DESCRIPTION
**Pull request by 64**
Adds our own copy of `strdup` (and also `strndup`) in `util.c`  because I couldn't be bothered trying to get travis to get it to work. Also a couple of other small makefile fixes which allows travis to work properly.

@bfgeshka could you go to https://travis-ci.org and enable travis for this repository? It would be much appreciated.

### CODE REVIEW

File: src/util.c [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#include "util.h"
+#include <stdlib.h>
+#include <string.h>
+#include <assert.h>
+
+char * tx_strdup(const char * s1)
+{
+	assert(s1 != NULL);
+	char * s2 = malloc(strlen(s1) + 1);
+	return s2 != NULL ? strcpy(s2, s1) : NULL;
```
Comment (desolt) [inline src/util.c]: nitpicking here, but memcpy would be fast here and we know the the exact number of bytes.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Termix
 **OpenGL powered terminal emulator for OSX and Linux**
 
+[![Build Status](https://travis-ci.org/bfgeshka/termix.svg?branch=master)](https://travis-ci.org/bfgeshka/termix)
```
Comment (desolt) [inline README.md]: Might be better to wait until it is actually enabled.

File: src/util.c (hunk 1 above)
Comment (64) [inline src/util.c]: Yeah, you're right.

File: README.md (hunk 2 above)
Comment (64) [inline README.md]: All the more motivation to fix it then!

**Pull request merged**